revproxy/views.py

```diff
@@ -20,9 +20,16 @@
 
 
 class ProxyView(View):
+    #: Variable to add remote_user to request
```

Comment:
The description can be better: for example here:

`Set to True if REMOTE_USER header should be added to the response in case the user is authenticated.`